docs/CHANGELOG.rst

```diff
@@ -28,6 +28,10 @@ Changed
 - Add ``source`` attribute to observer notifications
 - When subscribing to observer endpoint without explicit ids subscribe to CREATE and
   DELETE notifications by default
+- Send ``CREATE`` and ``DELETE`` notifications to the containers of the given object
```

Comment:
Make the commit message a bit more verbose.

Reply:
Done.